include/InstrumentFunctions.h

```diff
@@ -129,7 +129,7 @@ class InstrumentFunctionNoteStacking : public Model, public JournallingObject
 			ChordSemiTones m_semiTones;
 		};
 
-		static Init s_initTable[];
+		static std::vector<Init> s_initTable;
```

Comment:
I think this could just be a `std::array`, as it doesn't appear to be dynamically sized.